NineChroniclesUtilBackend/Controllers/AgentController.cs

```diff
@@ -0,0 +1,55 @@
+using Microsoft.AspNetCore.Mvc;
+using NineChroniclesUtilBackend.Models.Agent;
+using Bencodex.Types;
+using Libplanet.Crypto;
+using Microsoft.AspNetCore.Mvc;
+using Nekoyume;
+using Nekoyume.Action;
+using Nekoyume.Model.EnumType;
+using Nekoyume.Model.Item;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+using NineChroniclesUtilBackend.Models.Arena;
+using NineChroniclesUtilBackend.Services;
+using NineChroniclesUtilBackend.Arena;
+
+
+namespace NineChroniclesUtilBackend.Controllers;
+
+[ApiController]
+[Route("agent")]
+public class AccountController : ControllerBase
+{
+    [HttpGet("avatars/{agentAddress}")]
```

Comment:
It seems natural in the resource representation that the `agentAddress` comes first.

```suggestion
    [HttpGet("{agentAddress}/avatars")]
```